kernel/enter.c

```diff
@@ -765,6 +766,7 @@ int myst_enter_kernel(myst_kernel_args_t* args)
         &_main_thread, args->event, args->cwd, args->target_tid));
     thread = &_main_thread;
     process = thread->process;
+    myst_main_process = process;
```

Comment:
what is this variable used for?

Reply:
This is the first to be created and last process to go away. When we do a search for an active process in threads.c we now use this rather than the current process which could have already been zombified and so is no longer part of the active process list

Reply:
ah I see. I missed the references in thread.c.